packages/block-library/src/file/editor.scss

```diff
@@ -29,16 +23,27 @@
 		left: 0;
 	}
 
+	// This is an extra container present only in the editor view.
```

Comment:
Do we still need to have different markup in the editor?

Reply:
Probably not. The block feels somewhat oldish. Feel free to commit to the PR if you like!